PWGJE/FlavourJetTasks/AliAnalysisTaskHFJetIPQA.h

```diff
@@ -399,7 +399,7 @@ class AliAnalysisTaskHFJetIPQA: public AliAnalysisTaskEmcalJet
 
 
 
-    ClassDef(AliAnalysisTaskHFJetIPQA, 36)
+    ClassDef(AliAnalysisTaskHFJetIPQA, 37)
```

Comment:
ClassDef change would only be needed if in a previous commit data members were added and the ClassDef was not increased. Just change of the pointer type from derived to base class shouldn't require a ClassDef change.